frontend/interfaces/activity.ts

```diff
@@ -46,6 +46,8 @@ export enum ActivityType {
   EnabledMacOSSetupEndUserAuth = "enabled_macos_setup_end_user_auth",
   DisabledMacOSSetupEndUserAuth = "disabled_macos_setup_end_user_auth",
   TransferredHosts = "transferred_hosts",
+  EnableWindowsMdm = "enable_windows_mdm",
+  DisableWindowsMdm = "disable_windows_mdm",
```

Comment:
The name of the activities are `enabled_windows_mdm` and `disabled_windows_mdm` (past tense): https://github.com/fleetdm/fleet/issues/12288